pages/common/pveperf.md

```diff
@@ -0,0 +1,8 @@
+# pveperf
+
+> A benchmarking tool in proxmox server. Gather CPU/hard disk performance data on the hard disk mounted at PATH (/ is used as default).
```

Comment:
```suggestion
> A benchmarking tool in Proxmox Server. Gather CPU and hard disk performance data on the hard disk mounted at PATH (`/` is used as default).
```

Reply:
I don't think we need that much detail in the general description.
```suggestion
> A benchmarking tool in Proxmox Server. Gather CPU and hard disk performance data for the hard disk.
```